src/Steps.jsx

```diff
@@ -106,7 +108,7 @@ export default class Steps extends Component {
               return null;
             }
             const childProps = {
-              stepNumber: `${index + 1}`,
+              stepNumber: `${start + index}`,
```

Comment:
adding the prop`start` only to `stepNumber`, does not work prop `status` of component `Step`.

Reply:
`current` is still an index but not a step number.